docs/user-guide/cli-updatingcli.md

```diff
@@ -1,98 +1,155 @@
-# Updating Zowe CLI
+# Updating Zowe CLI and Zowe CLI plug-ins
 
-Zowe&trade; CLI is updated continuously. You can update Zowe CLI to a more recent version using online registry method or the local package method. 
+Zowe&trade; CLI is updated continuously. You can update Zowe CLI to a more recent version using either the online registry or the local package method.
+
+:::info Required role: systems administrator
+:::
 
 You must update Zowe CLI using the method that you used to install Zowe CLI.
 
-## Updating to the Zowe CLI V2 Long-term Support (v2-lts) version
+## Identifying the currently installed version of Zowe CLI and Zowe CLI plug-ins
+
+For Zowe CLI core, open a command line window and issue the following command:
+
+```
+zowe -V
+```
+
+For Zowe CLI plug-ins, open a command line window and issue the following command:
+
+```
+zowe plugins list
+```
+
+## Updating to the Zowe CLI V3 Long-term Support (v3-lts) version
 
-If you are running Zowe CLI version v1.8.x to v1.27.x, you can update to `@zowe-v2-lts` (LTS version) to leverage the latest Zowe CLI and plug-ins functionality.
+If you are running Zowe CLI version v2.0.0 to v2.15.x, you can update to `@zowe-v3-lts` (LTS version) to leverage the latest Zowe CLI and Zowe CLI plug-ins functionality.
```

Comment:
Please keep in mind that the version is not v2.x for the `zowe-v1-lts` tag
I guessing we were talking about Zowe Release versions here.
If so, we should probably state that here to avout confusion about the CLI version numbers that they got with the command above `zowe -V` (or `zowe--version`)